package.json

```diff
@@ -24,6 +24,7 @@
     "caniuse-lite": "^1.0.30001030",
     "electron-to-chromium": "^1.3.363",
     "node-releases": "^1.1.50"
+    "pkg-up": "^3.1.0"
```

Comment:
`pkg-up` requires `node >= 8` support. `browserslist` is depended by `@babel/preset-env` which still offers `node >= 6` support. Are there any chance that we may downgrade `pkg-up` to `^2.0.0`?

Reply:
Sure ae5e3a6

Reply:
Released at 4.11.1